content/source/docs/extend/community/contributing.html.md

```diff
@@ -66,8 +71,13 @@ play in the various areas of the code base.
 ### Issue Reporting Checklists
 
 We welcome issues of all kinds including feature requests, bug reports, and
-general questions. Below you'll find checklists with guidelines for well-formed
-issues of each type.
+general questions. The code and issue tracker for Terraform is at
```

Comment:
~general questions~ - I think we should drive people with questions to StackOverflow instead, as documented in https://www.terraform.io/community.html